doc/wg/core/notes/core-notes-2022-11-18.md

```diff
@@ -0,0 +1,222 @@
+   examples we want to preserve original headers but also have a copyright that
+   reflects the possibility for continued additions by Tock contributors.
+ * Amit: OK, our conclusion is that if they are different enough we would want
+   to modify we should create a new license, but the tool can be liberal about
+   accepting license that are close to our expectations to avoid needless
+   additional headers.
+ * Amit: What about files that do not support comments, e.g. markdown? The
+   current distinction of code vs. not code might be ambiguous.
+ * Amit: I proposed any file type that allows comments should have a license
+   header
+ * Hudson: How precise a category is that? Does a text file "allow" comments?
+ * Johnathan: We obviously should not require people to add metadata in PNGs --
+   are those comments?
+ * Phil: The statement says code artifacts
+ * Johnathan: That is vague
+ * Johnathan: I think license headers in all textual files that allow comments
+   should suffice
+ * Amit: Sounds good *adds that comment to the PR on his shared screen*
+ * Amit: I don't think that the license that applies to input that affects a
+   binary should carry the same license as documentation. Does it even make
+   sense to licenses for a README?
+ * Johnathan: Unless our documentation is under a different license things get
+   weird, and our documentation includes copied code.
+ * Amit: *updates comments on PR to reflect earlier discussion of not using RFC
+   language to specify actions of individuals*
+ * Leon: My next comment is that our current suggested approach feels weird
+   because it is different than what many other projects do with regards to
+   putting license headers in parent-module doc comments.
+ * Amit: I agree with Johnathan we should pick one or the other
+ * Leon: Sure, if so I think we should be careful the ambiguity of the text
+   does not allow either approach.
+ * Amit: Yeah, with a 1-3 line license header it seems that should go first.
+   This isn't a problem for the compiler right?
+ * Leon: Yeah, Rust allows comments above module documentation (but nothing
+   else)
+ * Phil: Sounds like Rust thought this through!
+ * Amit: I am gonna say that this is redundant since we are going to be more
+   liberal with code imported from third-parties.
+ * Johnathan: It sounds like we still want to follow a normal practice of
+   putting the header above module documentation:
+ * Amit: I will add that to the comment
+ * Amit: What do we do about tools? Two questions -- first, other repos that
+   are not the kernel, second, what about tools in the kernel repo (under
+   `tools/`). What if someone contributes a tool with a different license
+   header.
+ * Amit: A tricky bit is something that is AGPL, which is fine to use as a
+   tool, if it is not linked against.
+ * Johnathan: AGPL stands out as one license that Google's lawyers really do
+   not like, so they are overly conservative about it.
```

Comment:
`overly` -> `very`

I likely said something like "very conservative, perhaps overly conservative" that got shortened to "overly", but that's doesn't accurately represent my meaning.